cpp/tests/fixed_point/fixed_point_tests.cu

```diff
@@ -573,11 +573,14 @@ TYPED_TEST(FixedPointTestBothReps, NoScaleOrWrongTypeID)
 TYPED_TEST(FixedPointTestBothReps, SimpleFixedPointColumnWrapper)
 {
   using decimalXX = fixed_point<TypeParam, Radix::BASE_10>;
+  using RepType   = cudf::device_storage_type_t<TypeParam>;
 
-  auto const a = cudf::test::fixed_point_column_wrapper<int32_t>{{11, 22, 33}, scale_type{-1}};
-  auto const b = cudf::test::fixed_point_column_wrapper<int32_t>{{110, 220, 330}, scale_type{-2}};
+  auto const a = cudf::test::fixed_point_column_wrapper<RepType>{{11, 22, 33}, scale_type{-1}};
+  auto const b = cudf::test::fixed_point_column_wrapper<RepType>{{110, 220, 330}, scale_type{-2}};
+  auto const c = cudf::test::to_host<decimalXX>(a).first;
+  auto const d = cudf::test::to_host<decimalXX>(b).first;
 
-  CUDF_TEST_EXPECT_COLUMNS_EQUAL(a, b);
+  EXPECT_TRUE(std::equal(c.cbegin(), c.cend(), d.cbegin()));
```

Comment:
does "expect_columns_equivalent" not work here?

Reply:
I believe I attempted that and it failed. Let me give it a second look.

Reply:
there may need to be some modifications made to "expect columns equivalent" to treat fixed point columns with different scales but equivalent values as equivalent.

Reply:
Working now. I am sad that C++ doesn't have the `?` like LISPs and Ruby have. Would be nice to have `types_equivalent?`